Source/WebCore/platform/libwebrtc/LibWebRTCVPXVideoEncoder.cpp

```diff
@@ -307,7 +305,7 @@ webrtc::EncodedImageCallback::Result LibWebRTCVPXInternalVideoEncoder::OnEncoded
     }
 
     VideoEncoder::EncodedFrame encodedFrame {
-        Vector<uint8_t> { std::span { encodedImage.data(), encodedImage.size() } },
+        Vector<uint8_t> { unsafeMakeSpan(encodedImage.data(), encodedImage.size()) },
```

Comment:
Seems like we'd want to factor this out into its own function to be the safe interface to `webrtc::EncodedImage` rather than having it inline here. Not critical for this pull request.